lib/BloqadeLattices/Project.toml

```diff
@@ -3,6 +3,7 @@ uuid = "bd27d05e-4ce1-5e79-84dd-c5d7d508bbe4"
 version = "0.1.3"
 
 [deps]
+Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
```

Comment:
Actually I think we should get rid of this dependency because rand() can easily be used to sample from a uniform distribution:

'Uniform(a,b) = (a-b)*rand() + b'